client-session/README.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: pattern
+title: Client Session Pattern
+folder: client-session
+permalink: /patterns/client-session/
+categories: Client Architecture
+language: en
+tags:
+  - Client distributed
+  - Stateless servers
+---
```

Comment:
See the YAML frontmatter requirements in https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute

Please pick the category and the tags from the existing list, if applicable. See https://github.com/iluwatar/java-design-patterns/wiki/07.-Categories-and-Tags

Reply:
Fixed.